### PR TITLE
SEO-176113-Image-Alt-Text-Angular

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the image alt text.

Regards,
Beryl.

### CODE REVIEW

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Integration of Syncfusion Angular components into Angular Quick Start - Webpack
```
Comment (Yvone-Atieno) [inline angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:3]: Check title

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-description: Overview of Syncfusion Essential Angular.
+description: Learn here about Overview in Syncfusion Essential Angular Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:4]: Overview of

File: angular/GettingStarted/getting-started-with-visual-studio-environment.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title:  Getting Started with Angular on ASP.NET Core using Visual Studio 2015
-description: Overview of Syncfusion Essential Angular
+description: Learn here about overview in Syncfusion Essential Angular Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angular/GettingStarted/getting-started-with-visual-studio-environment.md:4]: Overview of

File: angular/GettingStarted/getting-started-with-visual-studio-environment.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title:  Getting Started with Angular on ASP.NET Core using Visual Studio 2015
```
Comment (Yvone-Atieno) [inline angular/GettingStarted/getting-started-with-visual-studio-environment.md:3]: Check title in all places

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md (hunk 2 above)
Comment (berylouma) [reply angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:4]: Updated

File: angular/GettingStarted/getting-started-with-visual-studio-environment.md (hunk 3 above)
Comment (berylouma) [reply angular/GettingStarted/getting-started-with-visual-studio-environment.md:4]: Updated

File: angular/GettingStarted/getting-started-with-visual-studio-environment.md (hunk 4 above)
Comment (berylouma) [reply angular/GettingStarted/getting-started-with-visual-studio-environment.md:3]: Updated

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md (hunk 1 above)
Comment (berylouma) [reply angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:3]: Updated

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+title: Getting started with Angular Platform | Syncfusion
+description: Learn here about Overview of Syncfusion Essential Angular Platform, its elements, and more.
 platform: Angular
 control: Introduction
 documentation: ug
 ---
 
 
-# Getting started with Angular Quick Start - Webpack
+# Getting started with Angular
```
Comment (Yvone-Atieno) [inline angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:11]: Revert this change

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Integration of Syncfusion Angular components into Angular Quick Start - Webpack
-description: Overview of Syncfusion Essential Angular.
+title: Getting started with Angular Platform | Syncfusion
```
Comment (Yvone-Atieno) [inline angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:3]: Getting Started with Angular Quick Start - Webpack | Syncfusion

File: angular/Tooltip/Getting-Started.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting started for Angular Tooltip
-description: How to create the Tooltip in Angular
+title: Getting started with Angular Tooltip Control | Syncfusion
+description: Learn here about getting started with Angular Tooltip in Syncfusion Essential Angular Tooltip Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angular/Tooltip/Getting-Started.md:4]: Learn here about getting started with Syncfusion Essential Angular Tooltip Control, its elements, and more.

File: angular/TreeView/Getting-Started.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Getting Started for Angular TreeView
-description: Getting Started for Angular TreeView
+title: Getting Started with Angular TreeView Control | Syncfusion
+description: Learn here about getting started with Angular TreeView in Syncfusion Essential Angular Treeview Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angular/TreeView/Getting-Started.md:3]: You have duplicated controlname and platformname. Please check in all places and use as above screenshot.
image

File: angular/TreeView/Getting-Started.md [hunk 9]
```diff
@@ -357,5 +357,5 @@ npm start
 
 * Browse to [http://localhost:3000](http://localhost:3000) to see the application. And navigate to TreeView tab. The component is rendered as like the below screenshot. You can make changes in the code found under src folder and the browser should auto-refresh itself while you save files. 
 
-![](Getting-Started_images/Getting-Started_img1.jpeg)
-
+![Running the systemJS application in Angular TreeView](Getting-Started_images/Getting-Started_img1.jpeg)
+
```
Comment (Yvone-Atieno) [inline angular/TreeView/Getting-Started.md:361]: Remove this extra space

File: angular/Slider/Getting-Started.md [hunk 10]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Getting Started for Angular Slider
-description: Getting Started for Angular Slider
+title: Getting Started with Angular Slider Control | Syncfusion 
+description: Learn here about getting started with Angular Slider in Syncfusion Essential Angular Slider Control, its elements, and more.
```
Comment (Yvone-Atieno) [inline angular/Slider/Getting-Started.md:3]: Check this

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md (hunk 5 above)
Comment (berylouma) [reply angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:11]: Done

File: angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md (hunk 6 above)
Comment (berylouma) [reply angular/GettingStarted/getting-started-with-angular-quick-start-webpack.md:3]: Done

File: angular/Tooltip/Getting-Started.md (hunk 7 above)
Comment (berylouma) [reply angular/Tooltip/Getting-Started.md:4]: Done

File: angular/TreeView/Getting-Started.md (hunk 8 above)
Comment (berylouma) [reply angular/TreeView/Getting-Started.md:3]: Done

File: angular/Slider/Getting-Started.md (hunk 10 above)
Comment (berylouma) [reply angular/Slider/Getting-Started.md:3]: Done

File: angular/TreeView/Getting-Started.md (hunk 9 above)
Comment (berylouma) [reply angular/TreeView/Getting-Started.md:361]: Done

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.
